canonical-web-and-design/maas-ui
# Response error json fallback

## Done
The API is inconsistent about returning errors as JSON, so we need to fallback to directly assigning errors that aren't parsable. 

## QA
* Attempt to delete the user with id: 2, this should result in an error which should endup in user state.

see: https://bugs.launchpad.net/maas/+bug/1840887
